src/test/java/org/fundacionjala/coding/SpinWordTest.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.coding;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by MarcandeaCabrera on 5/19/2017.
+ */
+public class SpinWordTest {
+
+    /**
+     *Verify words less than or equal to five characters.
+     */
+    @Test
+    public void wordLessOrEqualsThanFiveCharacters() {
+        SpinWord cad = new SpinWord();
+        assertEquals("Hola mundo", cad.reverseWord("Hola mundo"));
```

Comment:
This wrong
because the kata says : 
"but with all five or more letter words reversed"
Take a look again